cli/server/server.go

```diff
@@ -22,6 +22,7 @@ import (
 func NewCommands() []cli.Command {
 	var cfgFlags = []cli.Flag{
 		cli.StringFlag{Name: "config-path"},
+		cli.StringFlag{Name: "log-path", Usage: "file path for logging information (stdout if not given)"},
```

Comment:
Why not a configuration parameter?

Reply:
h,. What do you mean here?

Reply:
`protocol*.yml`

Reply:
added and set up higher priority for console input. BTW for config-path we don't have configuration in yml

Reply:
> BTW for config-path we don't have configuration in yml

That's natural, because you can't get a config until you know the `config-path`.